common/src/main/java/com/datastrato/gravitino/json/JsonUtils.java

```diff
@@ -240,6 +242,33 @@ public static ObjectMapper objectMapper() {
     return mapper;
   }
 
+  private static volatile ObjectMapper anyFieldMapper = null;
+
+  /**
+   * Get the shared ObjectMapper instance which can access all any field for JSON
+   * serialization/deserialization.
+   *
+   * @return The ObjectMapper instance.
+   */
+  public static ObjectMapper anyFieldMapper() {
+    if (anyFieldMapper == null) {
```

Comment:
I prefer https://blog.csdn.net/iteye_20954/article/details/81342188

Reply:
The double-check lazy mode should solve the memory model problem after the introduction of `volitile` in JDK 5, and currently only MySQL Backend will use the `Mapper`, so loading it separately is a more reasonable way

Reply:
The solution which I provided is lazy mode, too.

Reply:
Yes, I understand the solution you mentioned. The newly added `SqlSessionFactoryHelper` is loaded in this way, but this also means that the initialization is completed when the class is loaded. I think there is no need to worry about the initialization method. At present, we just keep the `Mapper` loading method the same as now

Reply:
After https://github.com/datastrato/gravitino/pull/2169 , `Mapper` has used the solution which I mentioned above.

Reply:
OK, I will change it to be consistent